packages/py-api/py_api/routes/routes.py

```diff
@@ -1,18 +1,24 @@
-from fastapi import APIRouter
-from py_api.routes.uploader_routes import UploaderRoutes
-from py_api.routes.url_shortener_routes import UrlShortenerRoutes
-from py_api.routes.utility_routes import UtilityRoutes
+from typing import List
+
+from fastapi import APIRouter, FastAPI
+from py_api.routes.uploader_routes import router as uploader_router
+from py_api.routes.url_shortener_routes import router as url_shortener_router
+from py_api.routes.utility_routes import router as utility_router
 
 """
     If you need to disable request verification for a particular endpoint,
     you can do that in packages/py-api/py_api/middleware/auth.py
     by adding them to the BYPASSED_ENDPOINTS dictionary.
 """
 
+routers: List[APIRouter] = [
+    uploader_router,
+    url_shortener_router,
+    utility_router,
+]
+
 
 class Routes:
-    @staticmethod
-    def bind(router: APIRouter) -> None:
-        UtilityRoutes.bind(router)
-        UploaderRoutes.bind(router)
-        UrlShortenerRoutes.bind(router)
+    def bind(app: FastAPI) -> None:
+        for router in routers:
+            app.include_router(router)
```

Comment:
Nice. We actually could get by without the class Routes and just have a bind function to which we pass the instance of our app (otherwise if we import it directly we would get circular import). But I prefer the OOP way, It keeps things organized and easier to manage.

Reply:
Yes, we can do that but it won't provide any more efficiency over this one. I guess this solely depends on personal preference. I would prefer to have the bind method encapsulated in a class rather than having it fly around. I bet you've had experienced at least once importing the "right" function from an incorrect module.

Reply:
Yes, totally agree.